DigitalLearningSolutions.Data/DataServices/UserDataService/DelegateUserDataService.cs

```diff
@@ -396,6 +397,26 @@ public void ActivateDelegateUser(int delegateId)
             );
         }
 
+        public int? GetDelegateUserLearningHubAuthId(int delegateId)

```

Comment:
This method was implemented in 681, but that has not yet been merged. As such, this method was stolen so that this ticket would function correctly.